lib/kbsecret/cli/kbsecret-rm

```diff
@@ -12,29 +12,33 @@ cmd = CLI.create do |c|
   c.slop do |o|
     o.banner = <<~HELP
       Usage:
-        kbsecret rm [options] <record>
+        kbsecret rm [options] <record [record ...]>
     HELP
 
     o.string "-s", "--session", "the session containing the record", default: :default
     o.bool "-i", "--interactive", "ask for confirmation before deleting"
   end
 
   c.dreck do
-    string :label
+    list :string, :labels
   end
 
   c.ensure_session!
 end
 
-label = cmd.args[:label]
+records = cmd.session.records :login
```

Comment:
Drop the `:login` here - we want *all* records, not just `login` typed ones. I think this is causing your test failures.